### ISSUE TITLE
The API doesn't work on IE11 browser because IE11 doesn't support ES6 proxy

### ISSUE THREAD

**Issue by tosun-si** [Vincent-Pang/builder-pattern#1]
I really appreciate the idea of this API. Thanks.
The problem is that the API doesn't work on IE11 browser because IE11 doesn't support ES6 proxy.

It should be a good thing if this API work on IE11.
What do you think about that ?

**Comment by Vincent-Pang** [comment]
I agree that it is a good thing if this API work on IE11. However, this library is heavily based on the ES6 proxy feature, and the ES6 proxy feature seems cannot be polyfilled according to the [StackOverflow question](https://stackoverflow.com/questions/45285992/es6-proxy-polyfill-for-ie11).

I have no idea how to make this library work on IE11. Do anyone have ideas?

**Comment by maeri** [comment]
Same here - today caught in this situation

**Comment by michaeldesigaud** [comment]
Congrats for this awesome library !

Maybe it could be interested to use this polyfill of ES6 Proxy.
https://github.com/GoogleChrome/proxy-polyfill
